docsets/Carbon/docset.json

```diff
@@ -1,14 +1,28 @@
 {
     "name": "Carbon",
-    "version": "1.30.0",
-    "archive": "Carbon-1.30.0.tgz",
+    "version": "1.31.0",
+    "archive": "versions/Carbon-1.31.0.tgz",
     "author": {
-        "name": "Brian Nesbitt",
-        "link": "https://github.com/briannesbitt"
+        "name": "Adnan RIHAN",
+        "link": "https://github.com/Max13"
     },
     "aliases": [
         "PHP Carbon",
         "Carbon DateTime",
         "PHP Carbon DateTime"
+    ],
+    "specific_versions": [
+      {
+        "version": "1.31.0",
+        "archive": "versions/Carbon-1.31.0.tgz"
+      },
+      {
+        "version": "1.30.0",
+        "archive": "versions/Carbon-1.30.0.tgz"
+      },
+      {
+        "version": "1.21.0",
+        "archive": "versions/Carbon-1.21.0.tgz"
```

Comment:
You need to actually add this archive. Moving the `txt` file doesn't do anything.

Reply:
These .tgz already exists, I had already submit them and they was processed. Does it mean, if I set specific_versions key, I have to resubmit the tgz? (Aren't they still in the CDN?)

Reply:
They are, but they're separate. Think of the main version and the specific_versions as completely separate. You don't need to re-add the archives once they were processed at that specific_version path, but if the path changes you need to re-add them.

Reply:
It should be fine now. My last commit sort it all out